collector/src/main/java/com/bitmovin/analytics/data/DebuggingEventDataDispatcher.kt

```diff
@@ -0,0 +1,33 @@
+package com.bitmovin.analytics.data
+
+import android.content.Context
+import com.bitmovin.analytics.BitmovinAnalyticsConfig
+import com.bitmovin.analytics.DebugCallback
+import com.bitmovin.analytics.license.LicenseCallback
+
+class DebuggingEventDataDispatcher(config: BitmovinAnalyticsConfig, context: Context, callback: LicenseCallback, private val debugCallback: DebugCallback) : IEventDataDispatcher {
+
+    private val innerEventDataDispatcher = SimpleEventDataDispatcher(config, context, callback)
```

Comment:
If we make this a constructor injected dependency then this class is stupid simple and we don't have to pass through all the constructor args for `SimpleEventDataDispatcher`